Command failed: npm access ls-collaborators X

```sh
❯ np --version
4.0.1

❯ np patch --yolo
  ❯ Prerequisite check
    ✔ Ping npm registry
    ✖ Verify user is authenticated
      → npm ERR! npm access edit [<package>]
      Check git remote
      Validate version
      Check for pre-release version
      Check git tag existence
    Git
    Bumping version using npm
    Publishing package using npm
    Pushing tags

✖ Command failed: npm access ls-collaborators supports-webp
npm ERR! 
npm ERR! Usage: This command is only available for scoped packages.
npm ERR! 
npm ERR! npm access public [<package>]
npm ERR! npm access restricted [<package>]
npm ERR! npm access grant <read-only|read-write> <scope:team> [<package>]
npm ERR! npm access revoke <scope:team> [<package>]
npm ERR! npm access 2fa-required [<package>]
npm ERR! npm access 2fa-not-required [<package>]
npm ERR! npm access ls-packages [<user>|<scope>|<scope:team>]
npm ERR! npm access ls-collaborators [<package> [<user>]]
npm ERR! npm access edit [<package>]
```

```sh
❯ npm i np@3.1 -g

❯ np --version
3.1.0

❯ np patch --yolo
  ✔ Prerequisite check
  ✔ Git
  ✔ Bumping version using npm
  ✔ Publishing package using npm
  ✔ Pushing tags

 supports-webp 1.0.7 published 🎉
```

I am also experiencing this. I went through the git history and it looks like this command was added in PR https://github.com/sindresorhus/np/pull/201. I haven't tried that commit yet, but I don't think this is `np`'s fault. Seems to be an npm bug.

```console
$ npx npm@6.5 access ls-collaborators
npx: installed 389 in 20.35s
{
  "sholladay": "read-write"
}

$ npx npm@6.6 access ls-collaborators
npx: installed 429 in 18.654s
npm ERR!
npm ERR! Usage: This command is only available for scoped packages.
npm ERR!
...
```


// cc @NeekSandhu

--- a/source/npm/util.js
+++ b/source/npm/util.js
@@ -39,11 +39,17 @@ exports.username = async ({externalRegistry}) => {
 	}
 };
 
-exports.collaborators = async packageName => {
+exports.collaborators = async pkg => {
+	const packageName = pkg.name;
 	ow(packageName, ow.string);
 
+	const args = ['access', 'ls-collaborators', packageName];
+	if (exports.isExternalRegistry(pkg)) {
+		args.push('--registry', pkg.publishConfig.registry);
+	}
+
 	try {
-		const {stdout} = await execa('npm', ['access', 'ls-collaborators', packageName]);
+		const {stdout} = await execa('npm', args);
 		return stdout;
 	} catch (error) {
 		// Ignore non-existing package error

--- a/source/prerequisite-tasks.js
+++ b/source/prerequisite-tasks.js
@@ -36,7 +36,7 @@ module.exports = (input, pkg, options) => {
 					externalRegistry: isExternalRegistry ? pkg.publishConfig.registry : false
 				});
 
-				const collaborators = await npm.collaborators(pkg.name);
+				const collaborators = await npm.collaborators(pkg);
 				if (!collaborators) {
 					return;
 				}

